### PR TITLE
Fix egress CIDR policy enforcement

### PR DESCRIPTION
**Pull request by tgraf**

### CODE REVIEW

**Comment by tgraf** [comment]
test-me-please

**Review by rlenglet — APPROVED** [review]

File: bpf/bpf_overlay.c [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/*
+ /*
```
Comment (rlenglet) [inline bpf/bpf_overlay.c]: Remove leading space.

**Review by rlenglet — APPROVED** [review]

**Comment by tgraf** [comment]
test-me-please

**Comment by tgraf** [comment]
test-me-please

**Comment by tgraf** [comment]
test-me-please

File: bpf/bpf_overlay.c (hunk 1 above)
Comment (tgraf) [reply bpf/bpf_overlay.c]: fixed

**Review by joestringer — APPROVED** [review]

**Comment by tgraf** [comment]
Fixed another bug that was hidden and added commit messages, doing another CI run.

**Comment by tgraf** [comment]
test-me-please

**Review by rlenglet — APPROVED** [review]

**Pull request merged**